modular_skyrat/modules/mold/code/mold.dm

```diff
@@ -0,0 +1,282 @@
+#define MAX_MOLD_FOAM_RANGE 7
+
+/datum/mold
```

Comment:
I'd probably rename that to `/datum/mold_type` just so it's clearer that it's mold types and not just straight up the mold itself.